x/oracle/keeper/native_token.go

```diff
@@ -21,7 +21,29 @@ import (
 // msg(refund or slash on beaconChain): update staker's price, operator's price
 
 const (
-	NSTETHASSETID = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x65"
+	// TODO: we currently support NSTETH only which has capped effective balance for one validator
+	// TODO: this is a bad practice, and for Lz, they have different version of endpoint with different chainID
+	// Do the validation before invoke oracle ralted functions instead of check these hard code ids here.
+	NSTETHAssetIDMainnet  = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x7595"
+	NSTETHAssetIDLocalnet = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x65"
+	NSTETHAssetIDHolesky  = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x9d19"
+	NSTETHAssetIDSepolia  = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x9ce1"
```

Comment:
Can we convert this to an enum? I know Go doesn't support string enums but maybe via `type NSTAssetID string` and `const ETHMainnet NSTAssetID = "0x...."` ?

Reply:
updated